Filters/PixelLevel/ShroomsFilter.cs

```diff
@@ -0,0 +1,44 @@
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace ComputerGraphics0.Filters.PixelLevel;
+
+public class ShroomsFilter : IImageFilter
+{
+    public string Name => "shrooms";
+    public Image<Argb32> Process(Image<Argb32> source)
+    {
+        var (avgR, avgG, avgB) = (0f, 0f, 0f);
+        var count = 0;
+
+        for (int i = 0; i < source.Width; i++)
+        {
+            for (int j = 0; j < source.Height; j++)
+            {
+                count += 1;
+                var pixel = source[i, j];
+                avgR += (avgR + pixel.R) / count;
+                avgG += (avgG + pixel.G) / count;
+                avgB += (avgB + pixel.B) / count;
+            }
+        }
```

Comment:
По идее тоже распараллелить можно, но тогда среднее надо просчитывать за циклом (хотя его и так за циклом считать надо)

```suggestion
        var (sumR, sumG, sumB) = (0, 0, 0);
        Parallel.For(0, source.Width, i =>
        {
            Parallel.For(0, source.Height, j=>
            {
                count += 1;
                var pixel = source[i, j];
                sumR += pixel.R;
                sumG += pixel.G;
                sumB += pixel.B;
            }
        });
        avgR = sumR/count;
        avgG = sumG/count;
        avgB = sumB/count;
```
Вот это примерный вариант, лучше проверить, чтобы он нормально работал, т.к. с телефона пишу

Reply:
Я думаю, лучше полностью избавиться от параллельных задач и не ебать себе мозг, нам скорость выполнения не принципиальна

Reply:
учитывая, что оно в целом работает - в принципе похер вообще, можно и так оставить. Тут просто именно что половина так, половина по-другому - доеб именно про это, чтобы единообразно было всё

Reply:
Ну чтобы единообразно было, давай только обычные for использовать

Reply:
ок